UM/VersionUpgradeManager.py

```diff
@@ -278,7 +283,7 @@ def _getFilesInDirectory(self, directory: str) -> Iterator[str]:
     ##  Gets all files that need to be upgraded.
     #
     #   \return A sequence of UpgradeTasks of files to upgrade.
-    def _getUpgradeTasks(self) -> Iterator[UpgradeTask]:
+    def _getUpgradeTasks(self) -> Iterator[FileTask]:
```

Comment:
I think this entire function should be renamed. It's not getting Upgrade Tasks anymore after all. (Also, the documentation is outdated)